### ISSUE TITLE
unrecognized ODF units: '(none)', 'psu' and 'kg/m**3'

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1063]
This is a random file. I will make a test suite that checks every ODF in this (private) assemblage of data.

```R
> library(oce)
Loading required package: gsw
Loading required package: testthat
> d<-read.oce("/data/flemishCap/moorings/1841/ProcessedMC/MCM_HUD2013021_1841_5830_300.ODF")
Warning messages:
1: In ODFNames2oceNames(ODFnames, ODFunits, PARAMETER_HEADER = NULL,  :
  unable to interpret ODFunits[3]='(none)', for item named 'time'

2: In ODFNames2oceNames(ODFnames, ODFunits, PARAMETER_HEADER = NULL,  :
  unable to interpret ODFunits[4]='(none)', for item named 'conductivity'

3: In ODFNames2oceNames(ODFnames, ODFunits, PARAMETER_HEADER = NULL,  :
  unable to interpret ODFunits[5]='psu', for item named 'salinity'

4: In ODFNames2oceNames(ODFnames, ODFunits, PARAMETER_HEADER = NULL,  :
  unable to interpret ODFunits[7]='kg/m**3', for item named 'sigmaTheta'
```

**Comment by dankelley** [comment]
I'm putting below some others I found in the Flemish Cap data.  First, the code that found them:
```R
library(oce)
options(warn=1) # print warnings as they occur
## I think this is all the ODFs in this dataset.
dirs <- c("/data/flemishCap/moorings/1840/ProcessedMC",
          "/data/flemishCap/moorings/1840/Processed_ADCP",
          "/data/flemishCap/moorings/1840/Processed_RCMnew",
          "/data/flemishCap/moorings/1841/ProcessedMC",
          "/data/flemishCap/moorings/1841/Processed_ADCP",
          "/data/flemishCap/moorings/1841/Processed_RCMnew",
          "/data/flemishCap/moorings/1842/ProcessedMC",
          "/data/flemishCap/moorings/1842/Processed_ADCP",
          "/data/flemishCap/CTD/2013/ODF",
          "/data/flemishCap/CTD/2013/QC",
          "/data/flemishCap/CTD/2014/ODF")
for (dir in dirs) {
    files <- list.files(path=dir, pattern="*.ODF", full.names=TRUE)
    for (file in files) {
        cat(file, "\n")
        d <- read.oce(file)
    }
}
```
Second, the new problems:
```
  odd unit (degrees) for 'directionTrue'; this will not affect calculations, though
  odd unit (mmho/cm) for 'directionMagnetic'; this will not affect calculations, though
```

**Comment by dankelley** [comment]
I've fixed all these up in `develop` commit 417449c9f12632fbafda05e933390f09d91008c2 and so I'll close this. I think my test catches all the ODF files in the Flemish Cap project, but if one of the other devels thinks I missed a directory in my code given in the [previous comment](https://github.com/dankelley/oce/issues/1063#issuecomment-241215084), I hope the will reopen this, which (I think) will trigger an email for me and I'll fix up the new problems. These problems are *very* easy for me to fix, because I know this code quite well.
